sdk/storage/azure-storage-queue/tests/test_queue_samples_authentication.py

```diff
@@ -75,22 +71,24 @@ def test_auth_active_directory(self):
 
         # Instantiate a QueueServiceClient using a token credential
         from azure.storage.queue import QueueServiceClient
-        queue_service = QueueServiceClient(account_url=self.url, credential=token_credential)
+        queue_service = QueueServiceClient(account_url=url, credential=token_credential)
         # [END create_queue_service_client_token]
 
         # Get information for the Queue Service
         properties = queue_service.get_service_properties()
 
         assert properties is not None
 
-    def test_auth_shared_access_signature(self):
+    @ResourceGroupPreparer()          
+    @StorageAccountPreparer(name_prefix='pyacrstorage')
+    def test_auth_shared_access_signature(self, resource_group, location, storage_account, storage_account_key):
         # SAS URL is calculated from storage key, so this test runs live only
-        if TestMode.need_recording_file(self.test_mode):
+        if not self.is_live:
             return
-
+        cs = self.connection_string(storage_account, storage_account_key)
```

Comment:
These are our samples - so a more descriptive name like "connection_string" might be user-friendlier

Reply:
updated...